### ISSUE TITLE
[vcpkg] support ports unit-tests

### ISSUE THREAD

**Issue by yurybura** [microsoft/vcpkg#11193]
It would be nice to have possibility to write a simple CMake based project to test a port.
This test must run on the CI machines after succeeded port build.

**Comment by strega-nil** [comment]
This is something we've discussed, but would take a lot of work to implement, and it's something that might make our CI take a lot longer to run. It's going to be a lot of discussion before we can implement this feature.

**Comment by BillyONeal** [comment]
This isn't something for our regular discussion meeting, dropping the tag.

We could add such a feature without necessarily turning it on in CI.

**Comment by JackBoosY** [comment]
Please track issue #72.

**Comment by strega-nil** [comment]
I think that this is a better place for discussion of port unit tests; #72 should be closed as we _have_ CI/CD to make sure that the ports work.
